account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
Then we could need one wizard developed in switzerland localization (PR open for v8).
https://github.com/OCA/l10n-switzerland/pull/162/files#diff-f006c2d78fe70d2df5e317db5b05160cR1

We could move this wizard inside this module. It is used to select some invoices and free them of a payment order (= remove them from the payment order and put them in a cancelled one).

This way, we better capture the workflow by showing the invoice was collected in a payment order but then it had a failure. If we simply allow collecting again the invoice, we cannot see that the first payment order was unsucessful.

So if you are OK with this, I can remove that functionality from the PR in swiss repository and include it in this PR.